src/components/desktop/Header.js

```diff
@@ -27,12 +30,37 @@ const StyledHeader = styled.header`
       li {
         display: inline-grid;
         place-items: center;
-        font-weight: bold;
         width: 110px;
         margin: 0;
         height: 100%;
-        color: ${({ theme }) => theme.background};
+        color: ${({ theme }) => theme.text};
         cursor: pointer;
+        position: relative;
+
+        a {
+          text-decoration: none;
+          color: ${({ theme }) => theme.text};
+          text-align: center;
+        }
+
+        .active {
+          background-color: ${({ theme }) => theme.secondaryBackground};
+          color: ${({ theme }) => theme.orange};
+          box-sizing: border-box;
+          padding-top: 25%;
```

Comment:
Is there a reason we're using a percentage for padding here? Where does this number come from?

Reply:
We got the percentage from a website we were using when designing the box-sizing and dropdown. I played around with the number and 25% is the ideal padding for the text to be centered and the boxes to be aligned with the top.